plugins/activity-resources/src/activity.ts

```diff
@@ -174,8 +179,53 @@ class ActivityImpl implements Activity {
     return !(collectionCUD && updateCUD) || ntx.objectId === object._id
   }
 
+  private readonly getUpdateTx = (tx: TxCUD<Doc>): [TxCollectionCUD<Doc, any>, TxUpdateDoc<Doc>] | undefined => {
+    if (tx._class !== core.class.TxCollectionCUD) {
+      return undefined
+    }
+
+    const colTx = tx as TxCollectionCUD<Doc, any>
+
+    if (colTx.tx._class !== core.class.TxUpdateDoc) {
+      return undefined
+    }
+
+    return [colTx, colTx.tx as TxUpdateDoc<Doc>]
+  }
+
   filterTxCUD (allTx: Array<TxCUD<Doc>>, hierarchy: Hierarchy): Array<TxCUD<Doc>> {
-    return allTx.filter((tx) => hierarchy.isDerived(tx._class, core.class.TxCUD))
+    const targetTxes = allTx.filter((tx) => hierarchy.isDerived(tx._class, core.class.TxCUD))
```

Comment:
I think you should filter entire transaction in case only one operation for this field is here, and filter individual operation inside TxView.svelte